src/app/pages/component-sidenav/_component-sidenav-theme.scss

```diff
@@ -1,36 +1,49 @@
 @import '../../../../node_modules/@angular/material/theming';
+@import '../../../styles/constants';
 
 @mixin component-viewer-sidenav-theme($theme) {
   $primary: map-get($theme, primary);
-  $accent: map-get($theme, accent);
-  $warn: map-get($theme, warn);
   $background: map-get($theme, background);
   $foreground: map-get($theme, foreground);
-  $sidenav: '.docs-component-viewer-sidenav';
+  $is-dark-theme: map-get($theme, is-dark);
+  $nav-background-opacity: if($is-dark-theme, 0.2, 0.03);
 
+  .docs-component-viewer-nav-inner {
```

Comment:
Is there a more descriptive name than `inner`, or a comment you could add to explain what the element is?